src/TicketMap/index.tsx

```diff
@@ -368,13 +370,38 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
 
   onClick = () => this.doSelect()
 
-  onTouchMove = () => this.setState({ isDragging: true })
+  onTouchStart = (event: React.TouchEvent<HTMLElement>) => {
+    if (event.changedTouches.length !== 1) {
+      return
+    }
+
+    const [ touchID ] = Object.getOwnPropertyNames(event.changedTouches)
```

Comment:
If you return unless there's only ever 1 `changesTouches` then can't we always assume touchId to be `"0"`?

Reply:
I suppose so. I just read up a bit more on the `TouchList` interface and it seems like you can get touches by index using `.item()`. Man, this is a strange interface. I'm wondering why `changedTouches` isn't simply an array...